.github/workflows/test_semeio.yml

```diff
@@ -56,4 +56,4 @@ jobs:
       run: |
         pushd semeio
         VIRTUAL_ENV=../.venv uv pip install ".[test]"
-        UV_PROJECT_ENVIRONMENT=../.venv uv run --no-sync pytest
+        UV_PROJECT_ENVIRONMENT=../.venv uv run --no-sync pytest --ert-integration
```

Comment:
Hm, I see the reason running them when building / testing komodo, but not as part of PRs. What's the reason then?